app/js/createUserObject.ts

```diff
@@ -77,38 +94,52 @@ function secondsToMinutes(time: number) {
 function findTestName(tests: Array<Test>, testId: number) {
     let testName = "None Assigned"
     tests.forEach(function(test) {
-        if (testId == test.id) {
+        if (testId === test.id) {
             testName = test.name
         }
-    })
+    });
     return testName
-}
+};
+
+function findCategoryName(categories: Array<Categories>, categoriesId: number) {
+    let categoriesName = "None Assigned"
+    categories.forEach(function(categories) {
```

Comment:
its good practise to use the singular in the forEach param, not the plural. as your not referring to one category